index.html

```diff
@@ -55,6 +55,157 @@ <h6>I'am a Software Developer</h6>
             <img class="Illustration" src="./Image/Illustration.png" alt="Illustration">
         </section>
 
+        <div class="work-section">
+            <div>
+                <h1 class="title">My Recent Works</h1>
+                <img class="vector-line" src="./Icons/Vector.png" alt="Black line">
+            </div>
+            <div>
+                <img class="Img-Placeholder" src="./Image/Img-Placeholder.png" alt="A lady in gym pose Image">
+                <h3 class="title-post">Multi-Post Stories</h3>
+                <p class="supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required. has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div>
+                    <ul class="tags">
+                        <li class="tag-1">css</li>
+                        <li class="tag-2">html</li>
+                        <li class="tag-3">bootstrap</li>
+                        <li class="tag-4">Ruby</li>
+                    </ul>
+                </div>
+                <button type="button"><span class="see-project">See Project</span></button>
+                <div class="project-1">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div>
+                        <ul class="tags">
+                            <li class="tag-p1">html</li>
+                            <li class="tag-p2">bootstrap</li>
+                            <li class="tag-p3">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
```

Comment:
it would be better if you aligned the content of every card project white `display: flex;`

Reply:
done